### PR TITLE
Minimal setup DesignSafe env files

### PR DESCRIPTION
**Pull request by rstijerina**
Adjusts the `designsafe.sample.env`, `mysql.sample.env` and `rabbitmq.sample.env` environment files for straightforward local development setup, accomplished by simply renaming the `*.sample.env` files to `*.env`.

e.g.
```
cp conf/env_files/designsafe.sample.env conf/env_files/designsafe.env
```

Steps to reproduce:

1. copy over sample env files, and edit the `MONGO_` settings in `designsafe.env` with the settings for the NCO database
2. `mkdir -p ~/corral-repl/tacc/NHERI`
3. `docker-compose -f conf/docker/docker-compose.yml build`
4. `docker-compose -f ./conf/docker/docker-compose-dev.all.debug.yml up`
5. `docker exec -it des_django bash`
6. `./manage.py migrate`
7. `./manage.py collectstatic -i demo`
8. `npm install`
9. `npm run dev`
10. (all steps related to accepting the ssl cert and adding `designsafe.dev` to `/etc/hosts`
11. visit designsafe.dev/nco and make frontend changes freely

### CODE REVIEW

**Review by SilversunKSauri — APPROVED** [review]
I've tested this out on both a Linux machine and a Windows machine and this branch works for both operating systems.  So looks like this is good.

File: README.md [hunk 1]
```diff
@@ -49,9 +50,10 @@ Machine, which is required to run Docker on Mac/Windows hosts.
 3. Set up local/testing database
 
    ```
-   $ docker-compose up -d
-   $ docker exec -it portal_django_1 bash
+   $ docker-compose -f ./conf/docker/docker-compose-dev.all.debug.yml up
+   $ docker exec -it des_django bash
    # ./manage.py migrate
+   # ./manage.py collectstatic
```
Comment (rstijerina) [inline README.md:56]:
```suggestion
   # ./manage.py collectstatic -i demo
```

File: README.md [hunk 2]
```diff
@@ -215,7 +217,8 @@ for testing.
 
 ```shell
 $ docker-compose build
```
Comment (rstijerina) [inline README.md:219]:
```suggestion
$ docker-compose -f conf/docker/docker-compose.yml build
```

File: README.md [hunk 3]
```diff
@@ -29,7 +29,8 @@ Machine, which is required to run Docker on Mac/Windows hosts.
 2. Build the container(s)
 
    ```
-   $ docker-compose build
+   $ docker-compose -f conf/docker/docker-compose.yml build
+   $ npm install
    ```
 
 3. Configure environment variables
```
Comment (SilversunKSauri) [inline README.md:35]: this needs to be for all 3 sample.env files, and it's sample.env, not env.sample

**Review by SilversunKSauri — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (fnets) [reply README.md:35]: Think I got it!

**Review by SilversunKSauri — DISMISSED** [review]
Readme looks good, env files are good.

File: README.md [hunk 4]
```diff
@@ -29,14 +28,15 @@ Machine, which is required to run Docker on Mac/Windows hosts.
...
 
-   Make a copy of [designsafe.env.sample](designsafe.env.sample) and rename it to
+   Make a copy of [designsafe.sample.env](designsafe.sample.env) and rename it to
```
Comment (rstijerina) [inline README.md:37]: @fnets can you fix the `designsafe.sample.env`, `rabbitmq.sample.env`, and `mysql.sample.env` links?

**Review by fnets — APPROVED** [review]
LGTM!

**Pull request merged**